src/main/java/build/buildfarm/worker/shard/Worker.java

```diff
@@ -286,7 +286,8 @@ public Worker(String session, ServerBuilder<?> serverBuilder, ShardWorkerConfig
         break;
     }
 
-    workerStubs = WorkerStubs.create(digestUtil);
+    workerStubs =
```

Comment:
We should have a default timeout in case the config is missing one.

Reply:
2 minute default if not configured